src/test/java/com/artipie/asto/S3StorageTest.java

```diff
@@ -59,6 +62,7 @@
  * Tests for {@link S3Storage}.
  *
  * @since 0.15
+ * @checkstyle MagicNumberCheck (500 lines)
```

Comment:
@olegmoz let's move `S3StorageTest` to `s3` package

Reply:
@Vatavuk agreed, done